modules/delegates.js

```diff
@@ -272,25 +281,30 @@ __private.checkDelegates = function (publicKey, votes, state, cb) {
  * @returns {setImmediateCallback} 
  */
 __private.loadDelegates = function (cb) {
-	var secrets;
+	var secretsList;
 
 	if (library.config.forging.secret) {
 		if (Array.isArray(library.config.forging.secret)) {
-			secrets = library.config.forging.secret;
+			secretsList = library.config.forging.secret;
 		} else {
-			secrets = [library.config.forging.secret];
+			secretsList = [library.config.forging.secret];
 		}
 	}
 
-	if (!secrets || !secrets.length) {
+	if (!secretsList || !secretsList.length || !library.config.forging.force) {
 		return setImmediate(cb);
 	} else {
-		library.logger.info(['Loading', secrets.length, 'delegates from config'].join(' '));
+		library.logger.info(['Loading', secretsList.length, 'delegates from config'].join(' '));
 	}
 
-	async.eachSeries(secrets, function (secret, cb) {
+	async.eachSeries(secretsList, function (encryptedItem, cb) {
+		var secret = __private.decryptSecret(encryptedItem.encryptedSecret, library.config.forging.defaultKey);
```

Comment:
I somewhat agree. I like the idea of enabling the forging of all delegates in before running tests.

But my understanding is that our main goal in this pull request is to disallow nodes to store plaintext passphrase for mainnet. By allowing plaintext passphrases in config (for testnets), and in turn, using them in `loadDelegates` function, we will be required to allow enable/disable forging to handle for secrets stored in plaintext. This overcomplicates a fairly simple feature for something we only want to use in testnets. Don't you think?